<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:url(:/images/bj.png)</string>
  </property>
  <widget class="QTabWidget" name="tw_page">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>350</width>
     <height>350</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page1">
    <attribute name="title">
     <string>  注册</string>
    </attribute>
    <widget class="QLineEdit" name="tel_edit_zc">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;输入8-11位手机号&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QLabel" name="mm_zc">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QLabel" name="tel_re_zc">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>    账号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mm_edit_zc">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;输入字母和数字穿插小于20位密码&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mm_edit_2zc">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;确认两次密码一致&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="mm_zc2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>101</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>  确认密码</string>
     </property>
    </widget>
    <widget class="QLabel" name="nc_zc">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>101</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>  昵称</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nc_edit_zc">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>190</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;输入8位以内昵称&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clear_zc">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>280</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QPushButton" name="commit_zc">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>280</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page2">
    <attribute name="title">
     <string>  登录</string>
    </attribute>
    <widget class="QPushButton" name="clear_dl">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>280</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QPushButton" name="commit_dl">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>280</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
    <widget class="QLabel" name="tel_re_dl">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>101</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string> 手机号</string>
     </property>
    </widget>
    <widget class="QLabel" name="mm_dl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tel_edit_dl">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>178</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>13945931550</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mm_edit_dl">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>1234567</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tel_edit_zc</tabstop>
  <tabstop>mm_edit_zc</tabstop>
  <tabstop>mm_edit_2zc</tabstop>
  <tabstop>nc_edit_zc</tabstop>
  <tabstop>clear_zc</tabstop>
  <tabstop>commit_zc</tabstop>
  <tabstop>tw_page</tabstop>
  <tabstop>tel_edit_dl</tabstop>
  <tabstop>mm_edit_dl</tabstop>
  <tabstop>clear_dl</tabstop>
  <tabstop>commit_dl</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
